### PR TITLE
pull request eip-3473.md

### PR DESCRIPTION
**Pull request by sgmfinance**
When opening a pull request to submit a new EIP, please use the suggested template: https://github.com/ethereum/EIPs/blob/master/eip-template.md

We have a GitHub bot that automatically merges some PRs. It will merge yours immediately if certain criteria are met:

 - The PR edits only existing draft PRs.
 - The build passes.
 - Your GitHub username or email address is listed in the 'author' header of all affected PRs, inside <triangular brackets>.
 - If matching on email address, the email address is the one publicly listed on your GitHub profile.

### CODE REVIEW

File: EIPS/eip-659.md [hunk 1]
```diff
@@ -0,0 +1,534 @@
+---
+eip: <659>
```
Comment (MicahZoltu) [inline EIPS/eip-659.md:2]:
```suggestion
eip: 3473
```
Filename needs to change too.

**Comment by sgmfinance** [comment]
I fixed all the issues but I cant find where is this 
eip-3473.md:	extra whitespace

File: EIPS/eip-3473.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+eip: 3473
+title: ERC-659 Multiple Callable Bonds Standard
+author: Yohji Sakamoto <info@sgm.finance>
+discussions-to: https://github.com/ethereum/EIPs/issues/3467
+type: Standards Track
+category : ERC
```
Comment (MicahZoltu) [inline EIPS/eip-3473.md:7]:
```suggestion
category: ERC
```

Comment (sgmfinance) [reply EIPS/eip-3473.md]: thx....

**Pull request closed without merging**